nrcatalogtools/catalog.py

```diff
@@ -69,6 +69,8 @@ def get(self, sim_name):
         metadata = self.get_metadata(sim_name)
         if type(metadata) is not dict and hasattr(metadata, "to_dict"):
             metadata = metadata.to_dict()
+        elif isinstance(metadata, dict):
+            metadata = dict(metadata.items())
```

Comment:
I am confused why this check is needed. Wouldn't `metadata` before and after this action be the same thing?

Reply:
This is just a quick fix for another small issue. Tbh, I think it could be addressed more appropriately from the `sxs` side. 

The issue is that somehow the type of `metadata` becomes the [`sxs.Metadata`](https://github.com/sxs-collaboration/sxs/blob/01667e6cd34/sxs/metadata/metadata.py#L26), and the [`update`](https://github.com/sxs-collaboration/sxs/blob/01667e6cd34/sxs/metadata/metadata.py#L612) method of which supports syntax like: `metadata.update(new_meta)`, but not `metadata.update(**new_data)`. 

But then the latter syntax is used in [here](https://github.com/sxs-collaboration/sxs/blob/01667e6cd34/sxs/time_series.py#L49). Since I am not sure whether `sxs` expects the `metadata` there has to be a pure `dict` or that they are not aware of this issue, I opt for this simple fix here. 

Going back to the original question, the [`sxs.Metadata`](https://github.com/sxs-collaboration/sxs/blob/01667e6cd34/sxs/metadata/metadata.py#L26) is a subclass from `OrderedDict`, so the `isinstance` and `items()` methods work, and converting it to a simple `dict` revert the `update` method back to the usual one.